tools/proto_format/BUILD

```diff
@@ -54,9 +56,20 @@ envoy_py_data(
     src = ":data_srcs",
 )
 
+pkg_files(
+    name = "protoprinted_files",
+    srcs = ["//tools/protoxform:api_protoxform"],
+    strip_prefix = strip_prefix.from_pkg(),
```

Comment:
this is pretty annoying - it used to be that to create a `pkg_tar` _and_ preserve dir structure you just gave it `srcs` now to do this you _must_ have 2 rules and this weird `strip_prefix` invokation